### PR TITLE
C major community version

### PR DESCRIPTION
**Pull request by airportyh**
# Why?

We should be tracking Clang on only it's major version.

https://releases.llvm.org/

### CODE REVIEW

File: modules.json [hunk 1]
```diff
@@ -1,90 +1,90 @@
 {
```
Comment (cdmistman) [inline modules.json:1]: Should this file be marked binary? It's just generated output right?

File: pkgs/modules/c/default.nix [hunk 2]
```diff
@@ -10,7 +10,7 @@ let
   dap-cpp = pkgs.callPackage ../../dap-cpp { };
   dap-cpp-messages = import ../../dap-cpp/messages.nix;
 
-  clang-version = lib.versions.majorMinor clang.version;
+  clang-version = lib.versions.major clang.version;
```
Comment (cdmistman) [inline pkgs/modules/c/default.nix:13]: Thought: should we have different modules for the different C standards? I'd ship C99 in its own module vs C11 vs...

**Review by cdmistman — APPROVED** [review]

File: pkgs/modules/c/default.nix (hunk 2 above)
Comment (airportyh) [reply pkgs/modules/c/default.nix:13]: I think that's a flag to clang. Maybe should be a configuration? But we don't have a way for users to configure things currently...

File: modules.json (hunk 1 above)
Comment (airportyh) [reply modules.json:1]: It can be mergeable by humans.

File: pkgs/modules/c/default.nix (hunk 2 above)
Comment (cdmistman) [reply pkgs/modules/c/default.nix:13]: gotcha, we should look into adding something for this...

**Pull request merged**